Contents/mods/Hydrocraft/media/scripts/Seeds ND.txt

```diff
@@ -360,6 +382,7 @@ item HCWheat
 	DisplayName		= Wheat,
 	Icon			= HCwheat,
 	Weight			= 0.009,
+		DisplayCategory	= SurFarm,
```

Comment:
hmmm not sure about this one. Maybe it shoud be foodR because it will be refined into food